rocm-comgr/PKGBUILD

```diff
@@ -21,6 +21,7 @@ build() {
         -DClang_DIR=/opt/rocm/lib/cmake/clang \
         -DLLD_INCLUDE_DIRS=/opt/rocm/include/lld \
         -DROCM_DIR=/opt/rocm/share/rocm/cmake \
+        -DCMAKE_SHARED_LINKER_FLAGS_INIT='-L/opt/rocm/lib -Wl,-rpath-link,/opt/rocm/lib' \
```

Comment:
Please use `CMAKE_EXE_LINKER_FLAGS` instead, `CMAKE_SHARED_LINKER_FLAGS_INIT` is meant to be set by toolchain configuration, not our use case here.